storybook/src/components/progress-indicator/progress-indicator.css-template.ts

```diff
@@ -17,11 +17,11 @@ export const cssProgressIndicator: ComponentImplementation<ProgressIndicator> =
             "dso-block": !!block,
           })}"
         >
-          <span
```

Comment:
Deze change is een BREAKING change: Afnemers moeten de gewijzigde markup voorschriften overnemen.

Dit component (Progress Indicator) zal nooit een ander icoon dan een spinner tonen. En als dat al zo zal zijn, dan bieden we dat aan met een variant of optie.

Maar het te tonen icoon is in ieder geval geen vrije keus. Soortgelijke situaties zijn de iconen in Alerts: De variant "Success" heeft altijd een groene checkmark, Info heeft altijd een blauwe I. De toolkit plaatst dit icoon vanuit de styling.

En daar hebben we een mooie faciliteit voor: De `di` mixins. DI staat voor DSO Icon.

Mijn voorstel is:
* De markup voorschrift voor de HTML/CSS implementatie terugdraaien.
* In `packages\dso-toolkit\src\components\progress-indicator\progress-indicator.scss` de di mixin toepassen:
  ```scss
  @use "../../di";

  .dso-progress-indicator-spinner {
    @include di.base("spinner");
  }
  ```

Dit geeft nos:

* Geen markup change ✅ 
* Geen vrije icon keuze ✅ 
* Eén bron voor het spinner icoon ✅ 

Los van bovenstaande punten: Dit had gewoon moeten werken. Ik vermoed dat het vanwege de styling komt, en dat onze spritesheet generator (dso-icons.svg) daar voor gebruik met `<svg><use href>` niet lekker op gaat 🤷

Reply:
Dank voor de hulp!